newenv/Include/Python.h

```diff
@@ -0,0 +1,140 @@
+#ifndef Py_PYTHON_H
```

Comment:
Please consider adding your virtual environment in the .gitignore file to avoid sending it with your codebase.

Reply:
The env file Ive added to the gitignore if you check but seems not to be ignored.